contents/docs/self-host/index.mdx

```diff
@@ -112,6 +112,8 @@ To upgrade, you can run the `upgrade-hobby` script from the PostHog repo.
 
 > **Warning:** Before upgrading, make sure you have created back-ups of all your data!
 
+Our recommendation is to keep your PostHog deployment up-to-date. While we avoid breaking changes wherever possible we may sometimes deprecate or add features that require you to update to the latest version.
```

Comment:
I'd add a link to the changelog too

Reply:
when you say require you to update - that's in reference to compatibility with our libraries, right ... as if they don't update their old instance itself will keep working just fine

Reply:
> when you say require you to update - that's in reference to compatibility with our libraries, right ... as if they don't update their old instance itself will keep working just fine

It's sort of purposefully vague.

It might be that an SDK changes enough that things stop working, or we want to deprecate something or change something. Really just trying to flag the expectation that step 1 of any support experience they have might be "can you update to latest?"

Reply:
> It's sort of purposefully vague.

Which might actually mean "it should be more clear" 🤣